AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -48,6 +51,19 @@ internal final class WrapperViewController: UIViewController {
                        completion: nil)
     }
 
+    private func updateTopScrollViewInsets(keyboardHeight: CGFloat,
+                                           preferredContentSize: CGSize,
+                                           finalHeight: CGFloat) {
+        guard keyboardHeight > 0 else { return }
+        let bottomScrollInset: CGFloat = preferredContentSize.height + keyboardHeight - finalHeight
+        topMostScrollView?.contentInset.bottom = bottomScrollInset
+    }
+
+    private lazy var topMostScrollView: UIScrollView? = {
```

Comment:
Maybe we shouldn't look for UIScrollView, but rather create protocol for "ScrollingContentViewController" or something and look for it?

Reply:
yeah, just did something similar.